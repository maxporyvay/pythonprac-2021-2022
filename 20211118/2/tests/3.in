class C:
	num = Num() 
print(C().num)
c, d = C(), C()
c.num = d.num = -1
print(c.num+d.num)
c.num = "qwerqwerqwer"
print(c.num+d.num)
d.num = range(10)
print(d.num)