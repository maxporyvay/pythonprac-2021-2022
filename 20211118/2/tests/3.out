0
-2
11
10
